src/gui/torrentmodel.h

```diff
@@ -33,15 +34,19 @@
 #define TORRENTMODEL_H
 
 #include <QAbstractListModel>
+#include <QColor>
```

Comment:
I was thinking that because they were being returned from functions they needed to be complete types. But you can declare functions which accept or return incomplete types. Rusty C++.

https://stackoverflow.com/questions/553682/when-can-i-use-a-forward-declaration